acquire/acquire.py

```diff
@@ -1111,6 +1111,16 @@ class History(Module):
                 ".config/BraveSoftware",
                 # Brave - MacOS
                 "Library/Application Support/BraveSoftware",
+                # Opera - Windows (Stable)
+                "AppData/Roaming/Opera Software/Opera Stable/Default",
+                "AppData/Local/Opera Software/Opera Stable/Default",
+                # Opera - Windows (GX)
+                "AppData/Roaming/Opera Software/Opera GX Stable",
+                "AppData/Local/Opera Software/Opera GX Stable",
+                # Opera - MacOS (Stable)
+                "Library/Application Support/com.operasoftware.Opera/Default",
+                # Opera - MacOS (GX)
+                "Library/Application Support/com.operasoftware.OperaGX",
```

Comment:
Could you add the some paths here to collect Opera browser artefacts for Linux based systems? This comment also applies to https://github.com/fox-it/dissect.target/pull/744.